src/Tribe/I18n.php

```diff
@@ -190,13 +253,25 @@ public function compile_translations( array $strings, $domains ) {
 
 				// Grab the possible strings for default and any other domain.
 				if ( 'default' === $domain ) {
-					$strings[ $key ][] = __( $value );
-					$strings[ $key ][] = __( strtolower( $value ) );
-					$strings[ $key ][] = __( ucfirst( $value ) );
+					if ($flags & static::COMPILE_INPUT) {
+						$strings[ $key ][] = __( $value );
```

Comment:
:no_entry_sign: The $text arg must be a single string literal, not "$value".

<sub>hash: 0c7f8aa2dce2a7a85a549c5077b803a2</sub>